pages/plugins.rst

```diff
@@ -6,22 +6,34 @@ Plugins
 
 General information
 ===================
-
-Graylog comes with a stable plugin API for the following plugin types:
-
-* **Inputs:** Accept/write any messages into Graylog
-* **Outputs:** Forward messages to other endpoints in real-time
-* **Services:** Run at startup and able to implement any functionality
-* **Alarm Callbacks:** Called when a stream alert condition has been triggered
-* **Filters:** Transform/drop incoming messages during processing
-* **REST API Resources:** A REST resource to expose as part of the ``graylog-server`` REST API
-* **Periodical:** Called at periodical intervals during server runtime
-* **Decorators:** Used during search time to modify the presentation of messages
-* **Authentication Realms:** Allowing to implement different authentication mechanisms (like single sign-on or 2FA)
+Graylog offers various extension points to customize and extend its functionality through writing Java code.
 
 The first step for writing a plugin is creating a skeleton that is the same for each type of plugin. The next chapter
 is explaining how to do this and will then go over to chapters explaining plugin types in detail.
 
+Plugin Types
+============
+
+ Graylog comes with a stable plugin API for the following plugin types:
```

Comment:
We should also mention that a single plugin can contain multiple of these types and not just one.